CONTRIBUTING.md

```diff
@@ -0,0 +1,217 @@
+[![master](https://travis-ci.org/checkpoint-restore/criu.svg?branch=master)](https://travis-ci.org/checkpoint-restore/criu)
+[![development](https://travis-ci.org/checkpoint-restore/criu.svg?branch=criu-dev)](https://travis-ci.org/checkpoint-restore/criu)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/55251ec7db28421da4481fc7c1cb0cee)](https://www.codacy.com/app/xemul/criu?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=xemul/criu&amp;utm_campaign=Badge_Grade)
+<p align="center"><img src="https://criu.org/w/images/1/1c/CRIU.svg" width="128px"/></p>
+
+## How to contribute to CRIU
+
+CRIU project is (almost) the never-ending story, because we have to always keep up with the
+Linux kernel supporting checkpoint and restore for all the features it provides. Thus we're
+looking for contributors of all kinds -- feedback, bug reports, testing, coding, writing, etc.
+Here are some useful hints to get involved.
+
+* We have both -- [very simple](https://github.com/checkpoint-restore/criu/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement) and [more sophisticated](https://github.com/checkpoint-restore/criu/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+feature%22) coding tasks;
+* CRIU does need [extensive testing](https://github.com/checkpoint-restore/criu/issues?q=is%3Aissue+is%3Aopen+label%3Atesting);
+* Documentation is always hard, we have [some information](https://criu.org/Category:Empty_articles) that is to be extracted from people's heads into wiki pages as well as [some texts](https://criu.org/Category:Editor_help_needed) that all need to be converted into useful articles;
+* Feedback is expected on the github issues page and on the [mailing list](https://lists.openvz.org/mailman/listinfo/criu);
+* We accept github pull requests and this is the preferred way to contribute to CRIU. If you prefer to send patches by email, you are welcome to send them to [CRIU development mailing list](https://lists.openvz.org/mailman/listinfo/criu).
+Below we describe in more detail recommend practices for CRIU developemnt.
+* Spread the word about CRIU in [social networks](http://criu.org/Contacts);
+* If you're giving a talk about CRIU -- let us know, we'll mention it on the [wiki main page](https://criu.org/News/events);
+
+### Seting up the developemnt environment
+
+Although criu could be run as non-root (see [Security](https://criu.org/Security), development is better to be done as root. For example, some tests require root. So, it would be a good idea to set up some recent Linux distro on a virtual machine.
```

Comment:
s/criu/CRIU/  (for consistency)

Reply:
criu could be run means the executable rather than project here, IMO ;-)
We could make it `criu` later on.